services/wallet/router/pathprocessor/processor_bridge_celar.go

```diff
@@ -364,6 +365,68 @@ func (s *CelerBridgeProcessor) sendOrBuild(sendArgs *MultipathProcessorTxArgs, s
 	return tx, nil
 }
 
+func (s *CelerBridgeProcessor) sendOrBuildV2(sendArgs *transactions.SendTxArgs, signerFn bind.SignerFn, lastUsedNonce int64) (*ethTypes.Transaction, error) {
```

Comment:
So all 4 `sendOrBuildV2` implementations are [not covered by tests](https://app.codecov.io/github/status-im/status-go/commit/da522e111b20d5320f4420b187df79870c163800/tree/services/wallet/router/pathprocessor). 

I understand there limitations, as anvil is not yet setup for bridging. 
But how can we be sure of the functionality state then? 🙁

Reply:
Actually what matter is `BuildTransactionV2` call, which is a replacement for `BuildTransaction` and it's implemented by all processors, some processors use `sendOrBuildV2`, called by `BuildTransactionV2` some not. The only `BuildTransactionV2` that is currently covered by tests is `BuildTransactionV2` implemented by the `TransferProcessor`. @antdanchenko did that just to unblock this PR, and will add more tests later to cover other flows.